src/test/regress/expected/multi_deparse_function.out

```diff
@@ -0,0 +1,311 @@
+--
+-- Regression tests for deparsing ALTER/DROP FUNCTION Queries
+--
+-- This test implements all the possible queries as of Postgres 11:
+-- 
+-- ALTER FUNCTION name [ ( [ [ argmode ] [ argname ] argtype [, ...] ] ) ]
+--     action [ ... ] [ RESTRICT ]
```

Comment:
can we add some tests with multiple actions to make sure those are correctly deparsed?

Reply:
I added a test for this here: https://github.com/citusdata/citus/blob/47adafe69adf4ad503e5f34f77cc5aec049115d7/src/test/regress/sql/multi_deparse_function.sql#L219